Changes in ASHE for specimin CI

NiharikaJamble opened the pull request:
In this change example.properties was renamed as config.properties for using it in CI for specimin

kelloggm reviewed (commented):
The only change in this PR appears to the addition of a newline at the end of the file. Is that intentionally the only change?

Based on our conversation, I was under the impression that there would be more here.

NiharikaJamble commented:
Hi Professor

The change was just change in file name.This was done as per instructions
provided by johnathan in the README file. I have adjusted the same in my
script and we won't be needing this PR.
Another reason I decided to go with a change in script over a PR is because
in case we go ahead with this PR , the file name change will keep showing
up in diff for future changes in this file.

I will close this PR.

Also I have found the cause of the script malfunctioning in my PR request
for CI integration in specimin project and fixed it. However, I am still
unable to actually run it in the PR request myself.

There was a difference in gradle build being used for running. Let me know
when can we connect to discuss how run issue


Thanks and regards
Niharika Jamble

On Fri, Jul 19, 2024 at 10:37 AM Martin Kellogg ***@***.***>
wrote:

> @ kelloggm commented on this pull request. The only change in this PR
> appears to the addition of a newline at the end of the file. Is that
> intentionally the only change? Based on our conversation, I was under the
> impression that there would be
> 
>
> ***@***.**** commented on this pull request.
>
> The only change in this PR appears to the addition of a newline at the end
> of the file. Is that intentionally the only change?
>
> Based on our conversation, I was under the impression that there would be
> more here.
>
> —
> Reply to this email directly, view it on GitHub
> <https://urldefense.com/v3/__https://github.com/njit-jerse/ASHE_Automated-Software-Hardening-for-Entrypoints/pull/1*pullrequestreview-2188399604__;Iw!!DLa72PTfQgg!NThtbZu4BuHHJisMoi3Ox8LeJADw24I3MipMKkrs4xLmrAPwcrMEEhPZrvRo-eSdtSxL9kGlcRCcdJWF3h-qwbxk3w$>,
> or unsubscribe
> <https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/BIZGHND2WHJWDGSSKCGL6ETZNEQDXAVCNFSM6AAAAABLBQMZ7GVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDCOBYGM4TSNRQGQ__;!!DLa72PTfQgg!NThtbZu4BuHHJisMoi3Ox8LeJADw24I3MipMKkrs4xLmrAPwcrMEEhPZrvRo-eSdtSxL9kGlcRCcdJWF3h-Da0YQUg$>
> .
> You are receiving this because you authored the thread.Message ID:
> <njit-jerse/ASHE_Automated-Software-Hardening-for-Entrypoints/pull/1/review/2188399604
> @github.com>
>

kelloggm commented:
> Also I have found the cause of the script malfunctioning in my PR request
for CI integration in specimin project and fixed it. However, I am still
unable to actually run it in the PR request myself.

See my comment on https://github.com/njit-jerse/specimin/pull/323

> Let me know when can we connect to discuss how run issue

My calendar says we have a meeting scheduled for this afternoon at 4:45pm
